src/python/pants/backend/url_handlers/s3/integration_test.py

```diff
@@ -121,69 +139,149 @@ def new_init(self, **kwargs):
 
 
 @pytest.mark.parametrize(
-    "request_url, expected_auth_url, expected_native_url, req_type",
+    "request_url, expected_auth_url, expected_native_url, req_type, auth_type",
     [
         (
             "s3://bucket/keypart1/keypart2/file.txt",
+            "https://s3.amazonaws.com/bucket/keypart1/keypart2/file.txt",
```

Comment:
under hmacv1 the url thats signed is the s3 path style (which can be different that what the request is actually made with)